js/src/services/project-api.ts

```diff
@@ -31,7 +31,7 @@ export const fetchProject = (slug: string) => {
       success: true,
       data: resp.data,
     }))
-    .catch<Failure>(() => ({ success: false, reason: "Error" }));
+    .catch<Failure>(() => ({ success: false, reason: "something_wrong" }));
```

Comment:
Your `Something went wrong` reason everywhere the different.

Reply:
Done